constraints.txt

```diff
@@ -58,18 +59,19 @@ executing==2.0.1
 fastjsonschema==2.19.1
 fastobo==0.12.3
 filelock==3.13.1
-fonttools==4.49.0
+flake8==7.0.0
```

Comment:
There seem to be quite a few "developer" dependencies coming in this time..

Reply:
Again, some package dependency list must have added this as a main dependency rather than a dev one.